Dockerfiles/Dockerfile.frontend

```diff
@@ -6,3 +6,4 @@ WORKDIR /web/frontend
 COPY ../web/frontend .
 RUN npm install
 ENTRYPOINT ["npm", "start"]
+CMD []
```

Comment:
Remove empty `CMD`

Reply:
I suggest removing the entrypoint, and keeping the CMD. this way, we can override the command from "npm start" to "npm run dev" in the compose file (for development).

Reply:
isn't the idea to override the entrypoint and using cmd for additional arguments for the entrypoint?

Reply:
> I suggest removing the entrypoint, and keeping the CMD. this way, we can override the command from "npm start" to "npm run dev" in the compose file (for development).

wouldn't the "proper" way be

```
ENTRYPOINT ["npm"]
CMD ["start"]
```
and override CMD with "run dev"?